config/routes.rb

```diff
@@ -6,5 +6,7 @@
 
   post '/', to: 'answers#create'
   match '*path', to: 'answers#create', via: :post
-  match '*path', to: 'pages#show', via: :all
+  match '*path', to: 'pages#show',
+    via: :all,
+    constraints: lambda {|req| req.path !~ /\.(png|jpg|js|css|ico)$/ }
```

Comment:
this does make me cry some 😢

Reply:
When we serve via CDN this won't be needed.